src/stylesheets/app.scss

```diff
@@ -0,0 +1,298 @@
+@import "base/vars";
+@import "base/typography";
+
+body {
+  background-color: $florence-light-blue; // only temporary
+  margin: 0;
+}
+
+ul {
+  padding: 0;
+}
+
+sup {
+  color: #e95e67;
+  text-decoration: underline;
+}
+
+/*-------- Top Fold styles ---------*/
+
+#top-fold {
+  background: $florence-light-blue;
+  overflow: hidden;
+  padding-top: 50px;
+  padding-bottom: 15%; //only temporary
+
+  #title-box {
+    position: relative;
+  }
+
+  #title {
+    margin: auto;
+    max-width: 50%;
+    @media (max-width: $screen-sm){
+      margin-bottom: 8%;
+      max-width: 85%;
+    }
+  }
+
+  #subtitle {
+    color: $florence-blue-green;
+    font-family: 'Myriad Pro';
+    font-size: 14px;
+    letter-spacing: 3px;
+    margin-bottom: 2%;
+    text-align: center;
+    text-transform: uppercase;
+  }
+
+  #yellow-pill {
+    left: 22%;
+    position: absolute;
+    top: -30%;
+    transform: rotate(35deg);
+    width: 7%;
+
+    @media (max-width: $screen-sm) {
+      width: 20%;
+      top: -55%;
+      left: 1%;
+      transform: rotate(40deg);
+    }
+  }
+
+  #pills {
+    left: 12%;
+    position: absolute;
+    top: 65%;
+    width: 10%;
+
+    @media (max-width: $screen-sm) {
+      display: none;
+    }
+  }
+
+  #more-pills {
+    position: absolute;
+    right: 12%;
+    top: -20%;
+    transform: rotate(10deg);
+    width: 20%;
+
+    @media (max-width: $screen-sm) {
+      right: -35%;
+      top: -40%;
+      width: 60%;
+    }
+  }
+
+  #purple-pills{
+    position: absolute;
+    right: 10%;
+    width: 12%;
+
+    @media (max-width: $screen-sm) {
+      bottom: -30%;
+      right: -22%;
+      width: 35%;
+      z-index: 4;
+    }
+  }
+}
+
+/*----------- Pillbar styles ----------*/
+
+.pillbar {
+  background-image: linear-gradient(0deg, $florence-red 75%, $florence-blue 25%);
+  background-image: -webkit-linear-gradient(0deg, $florence-red 75%, $florence-blue 25%);
+  border-radius: 70px;
+  box-shadow: inset 2px -15px 2px rgba(0, 0, 0, 0.1);
+  color: #d2e7e2;
+  display: flex;
+  font-family: 'geomanist';
```

Comment:
When you implement some more sass variables, having some for your geomanist fonts would be helpful. Also might want to add a fallback font. Just `sans-serif` would probably be fine, so `font-family: 'geomanist, sans-serif'`, or `font-family: $whatever-your-font-variable-name-is`.